src/gui/tray/Window.qml

```diff
@@ -403,67 +403,59 @@ Window {
                     }
                 }
 
-                // Filler between account dropdown and header app buttons
-                Item {
-                    id: trayWindowHeaderSpacer
-                    Layout.fillWidth: true
-                }
-
-                HeaderButton {
-                    id: openLocalFolderButton
+                RowLayout {
+                    id: openLocalFolderRowLayout
+                    height: Style.trayWindowHeaderHeight

```

Comment:
This is not necessary and is discouraged. Please use only implicitWidth or Layout.preferredWidth .